recipes/jaxopt/meta.yaml

```
@@ -0,0 +1,48 @@
+{% set name = "jaxopt" %}
+{% set version = "0.5" %}
+{% set sha256 = "9d48c30c78285ec5beea0264bb21fa69bbf9b82354136f0aaab0f956f82dd797" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz  
+  sha256: {{ sha256 }}
```

Don't worry about templatizing the hash, just put it in here:
```suggestion
  sha256: 9d48c30c78285ec5beea0264bb21fa69bbf9b82354136f0aaab0f956f82dd797
```